src/setup.js

```diff
@@ -42,6 +42,18 @@ const setupHooks = () => {
 			return editorProps;
 		}
 	);
+
+	// Hook to expand the supported endpoints of `api-fetch` library.
+	addFilter(
+		'native.supported_endpoints',
+		'gutenberg-mobile',
+		( endpoints ) => {
+			return {
+				...endpoints,
+				POST: [ ...endpoints.POST, /wpcom\/v2\/(media)\/.*/i ],
```

Comment:
The `wpcom/v2/media` endpoint will be used in the VideoPress block ([reference](https://github.com/Automattic/jetpack/pull/29756/files#diff-5d8b68641e714ee87db736b00a627212fce17c3336abaff2828d5321edb19695R46)).